### ISSUE TITLE
Draw the loss curve

### ISSUE THREAD

**Issue by 943fansi** [xuhongzuo/couta#6]
@xuhongzuo . I want to use `pd.DataFrame` to save `loss`, `loss_oc` and `val_loss` in each epoch, and save it as a csv file. Could you add the code for this function?

**Comment by xuhongzuo** [comment]
I printed these loss values after each training epoch.   
You could do this personalized requirement by yourself. 
issue closed.

**Comment by 943fansi** [comment]
Add 4 lines code at `src/algorithms/couta_algo.py`, can save info in a "train_log.txt" file.
```
...
    val_loss = torch.mean(torch.stack(val_loss)).data.cpu().item()
# start
loss_file = "train_log.txt"
_f = open(loss_file, "a")
print(f'epoch: {i+1:02},loss: {epoch_loss:.6f},loss_oc: {epoch_loss_oc:.6f}, val_loss: {val_loss:.6f}',
                  file=_f, flush=True)
_f.close()
#end
    if (i+1) % 10 == 0:
    ...
```
And then use following code to transform it into a csv file:
```
import numpy as np
import pandas as pd

with open("train_log.txt", "r") as f:
    data = f.readlines()

n = len(data)
result = np.zeros([n, 4])
for i in range(n):
    record = data[i].split(",")
    epoch = int(record[0].split(" ")[-1])
    loss = float(record[1].split(" ")[-1])
    loss_oc = float(record[2].split(" ")[-1])
    val_loss = float(record[3].split(" ")[-1])
    print(epoch, loss, loss_oc, val_loss)
    result[i, :] = [epoch, loss, loss_oc, val_loss]
df = pd.DataFrame(data=result, columns=["epoch", "loss", "loss_oc", "val_loss"])
df.to_csv("loss_curve.csv")
```
